release.py

```diff
@@ -0,0 +1,488 @@
+import base64
+import getpass
+import json
+import os
+import re
+import subprocess
+import urllib2
+
+try:
+    from setuptools import Command
+except ImportError:
+    from distutils.core import Command
+
+GITHUB_REPOSITORY_API = 'https://api.github.com/repos/prestodb/presto-admin'
+CURRENT_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
+
+
+class SemanticVersionHelper(object):
+    def __init__(self, version):
+        self.version = version
+        self._set_semantic_version_fields()
+
+    @staticmethod
+    def _get_version_field_value(version_fields, index):
+        try:
+            return version_fields[index]
+        except IndexError:
+            # The field value was omitted for the version
+            return 0
+
+    def _set_semantic_version_fields(self):
```

Comment:
Nit: this should probably be inline in the constructor. It isn't used elsewhere, and it's weird that it operates by side-effect like this.